.github/workflows/ci.yml

```diff
@@ -1,8 +1,6 @@
 name: CI
 
 on:
-  schedule:
```

Comment:
This workflow runs unit and integration tests.

Reply:
But could turn red because of external dependencies, which we wouldn't notice in periods with little or no PRs.